### PR TITLE
Add GitHub action to build images

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: java/images/centos-java11/README.md [hunk 1]
```diff
@@ -118,6 +118,12 @@ Example:
 export MAVEN_OPTS="$(run-java.sh options --memory)"
 mvn clean install
 ```
+
+
+undefined
```
Comment (JohnNiang) [inline java/images/centos-java11/README.md:123]: `undefined` may be clerical error.

File: java/images/centos/README.md [hunk 2]
```diff
@@ -119,6 +119,11 @@ export MAVEN_OPTS="$(run-java.sh options --memory)"
 mvn clean install
 ```
 
+
+undefined
```
Comment (JohnNiang) [inline java/images/centos/README.md:123]: `undefined` may be clerical error.

File: .github/workflows/pull-request.yaml [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+      - 'master'
+
+jobs:
+  Test:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dep
+      run: |
+        curl -L https://github.com/linuxsuren/http-downloader/releases/latest/download/hd-linux-amd64.tar.gz | tar xzv
```
Comment (JohnNiang) [inline .github/workflows/pull-request.yaml:20]: We should use `http-downloader` with version specified here instead of latest.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request.yaml [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+      uses: docker/build-push-action@v2.4.0
+      if: github.repository_owner != 'kubesphere'
+      with:
+        context: java/images/centos-java11
+        tags: ${{ steps.metaContributors.outputs.tags }}
+        push: ${{ github.event_name != 'pull_request' }}
+        labels: ${{ steps.metaContributors.outputs.labels }}
+        platforms: linux/amd64,linux/arm64
+
+  BuildCentOS:
```
Comment (JohnNiang) [inline .github/workflows/pull-request.yaml:102]: BuildCentOs is ambiguity, and I suggest BuildCentOsJava8 is clearer.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: java/images/centos-java11/README.md (hunk 1 above)
Comment (LinuxSuRen) [reply java/images/centos-java11/README.md:123]: This file was generated by `fish-pepper`.

File: java/images/centos/README.md (hunk 2 above)
Comment (LinuxSuRen) [reply java/images/centos/README.md:123]: This file was generated by fish-pepper.

File: .github/workflows/pull-request.yaml (hunk 3 above)
Comment (LinuxSuRen) [reply .github/workflows/pull-request.yaml:20]: Good point.

File: .github/workflows/pull-request.yaml (hunk 4 above)
Comment (LinuxSuRen) [reply .github/workflows/pull-request.yaml:102]: Excellent suggestion.

**Pull request merged**